sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -82,6 +82,15 @@ export async function executeAtomXmlOperation(
  */
 export function serializeToAtomXmlRequest(resourceName: string, resource: any): object {
   const content: any = {};
+
+  // Remove key value pairs that have value as undefined. This is because the Service Bus ATOM based management API
+  // does not accept empty XML elements in the top level request, but accepts empty elements within some of the inner properties
+  Object.keys(resource).forEach(function(property) {
```

Comment:
If possible, I think it would be better to do this outside of `serializeToAtomXmlRequest` (perhaps in another helper), else make a copy with the undefined property keys removed. It's risky to modify the incoming object as callers may not be expecting a serialize method to permanently modify the object being serialized.

Reply:
Updated, although the object is actually created internally based on user inputs.